tests/integration/targets/nxos_zone_zoneset/tests/common/sanity.yaml

```diff
@@ -144,13 +144,19 @@
       register: result
       cisco.nxos.nxos_zone_zoneset: *id001
 
+    - name: Disaplay Idempotence Check result
+      debug:
+        var: result
+
     - assert:
         that:
-          - result.commands == []
+          - result.commands == ["terminal dont-ask", "zoneset activate name zsetname1
```

Comment:
Why is an idempotent task sending commands?

Reply:
zoneset activation is a bit unique it needs to be sent always